packages/core/src/utils/logger.ts

```diff
@@ -5,6 +5,7 @@
 import chalk from 'chalk';
 import { PipcookRunner } from '../runner';
 import { PipcookComponentResult } from '../types/component';
+const debugLog = require('debug')('core/utils');
```

Comment:
As I mentioned before, debug and logger have different users, debug is for developer, and the log is for user.